test/built-ins/RegExp/prototype/Symbol.replace/y-fail-return.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Stops match execution after first match failure
+es6id: 21.2.5.8
+info: >
+    21.2.5.8 RegExp.prototype [ @@replace ] ( string, replaceValue )
+
+    [...]
+    13. Repeat, while done is false
```

Comment:
Similar to my comment about the test case for the existence of the **global** flag with **Symbol.match**, it would provide additional coverage if we also have a test case for **Symbol.replace**.

An example assertion would be `assert.sameValue(/a/gy[Symbol.replace]('aaba', 'c'), 'ccba')`.